tash

```diff
@@ -0,0 +1,31 @@
+[1mdiff --git a/src/foraging_gui/Foraging.py b/src/foraging_gui/Foraging.py[m
```

Comment:
What is the tash file?

Reply:
After talking with Micah, this file seems to be added by mistake. @micahwoodard Could you make a PR to delete it?